train-tracker/src/styles/Search.css

```diff
@@ -101,4 +101,14 @@
 
 .magenta-icon {
     color: #ed156f;
+}
+
+/*Between 768 and 1050, buttons on search should go vertical*/
+@media (min-width:768px) and (max-width:1050px) {
```

Comment:
imo we should just make the components go vertical sooner rather than having the search component go very small at this specific range. not too huge either way so your choice

Reply:
Okay i made the search components go vertical as well, but the buttons specifically fall out of the form this makes sure they don't fall out but don't go vertical on the mobile view